data_safe_haven/external/api/graph_api.py

```diff
@@ -312,19 +312,19 @@ def create_application_secret(
             raise DataSafeHavenMicrosoftGraphError(msg) from exc
 
     def create_group(self, group_name: str) -> None:
-        """Create an AzureAD group if it does not already exist
+        """Create an Entra ID group if it does not already exist
```

Comment:
```suggestion
        """Create an Entra group if it does not already exist
```

Reply:
As per https://learn.microsoft.com/en-us/entra/fundamentals/how-to-rename-azure-ad, a group is a feature, so it's `Entra group`